llvm/test/CodeGen/AMDGPU/mcexpr-knownbits-assign-crash.ll

```diff
@@ -0,0 +1,120 @@
+; REQUIRES: asserts
+; RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx1030 -o - %s 2>&1 | FileCheck %s
+
```

Comment:
Add a comment explaining what this tests. Include the issue number in the filename?